application/src/main/java/team/asd/service/PaymentTransactionService.java

```diff
@@ -44,9 +44,6 @@ public List<PaymentTransaction> readByReservationIdStatus(Integer reservationId,
 	}
 
 	public List<PaymentTransaction> readByChargeTypePartnerIdFundsHolderStatus(String chargeType, Integer partnerId, Integer fundsHolder, String status) {
-		if (ObjectUtils.anyNull(fundsHolder, status)) {
```

Comment:
`partnerId` parameter should be required
It's not enough controller (@PathVariable) validation, because you can use service method in other services

Reply:
fixed